docs/browser.md

```diff
@@ -78,3 +78,46 @@ to help us visualize the imports and determine which module is importing a certa
       ![Dependency Graph](./images/dependency-graph.svg)
     - Additionally specify a certain dependency with `--reaches` , `npx depcruise src/srcShared/fs.ts --reaches "fs-extra" --output-type dot | dot -T svg > dependency-graph.svg`, to hide unrelated dependencies:
       ![Dependency Graph](./images/dependency-graph-small.svg)
+
+## Global Scoped Objects + Testing behavior
+
+### Summary
+
+-   **When in Browser mode**, state is not shared between the actual extension code and the unit test code. I.e you cannot modify a global variable in the extension code and see that change in the unit tests
+-   State will need to be stored somewhere in `globalThis` to be accessible by tests. Any state not in `globalThis` will not be the same as the actual extension, they are separate.
+
+With the introduction of Browser support, it was discovered that the context between the extension code and test code is not shared.
+Though it is shared in the Node version of the extension.
+
+Example:
+
+```typescript
+// extension.ts
+export let myGlobal = 'A'
+
+function activate() {
+    // Change the global variable value
+    myGlobal = 'B'
```

Comment:
this is the "wrong" way, need an example of the "right" way?

Reply:
Added a right way example